package.json

```diff
@@ -35,6 +35,7 @@
     "antd": "4.23.1",
     "axios": "0.24.0",
     "babel-preset-react-app": "3.1.2",
+    "classnames": "^2.3.2",
```

Comment:
After some refactoring where I removed the conditionality of the html classes, I didn't really need this library for this PR anymore, but I'm sure we will in the future so I left it. It is very handy.